### ISSUE TITLE
nuklear_combo

### ISSUE THREAD

**Issue by exchg** [Immediate-Mode-UI/Nuklear#198]
Looks like there is some inconsistency in this commit: 1e21ef22856bf5734bb280be6fb88f4687656a46               
Author:     Michael Aganier <michaelaganier@gmail.com> 
AuthorDate: Fri Sep 4 08:15:01 2020 -0400              
Commit:     Michael Aganier <michaelaganier@gmail.com> 
CommitDate: Fri Sep 4 08:29:43 2020 -0400              

Thisi is function definition and 'selected' defined as boolean
``` c++
NK_API nk_bool
nk_combo_separator(struct nk_context *ctx, const char *items_separated_by_separator,
    int separator, nk_bool selected, int count, int item_height, struct nk_vec2 size)
{
```
and this is how 'selected' used inside

``` c++
    /* find selected item */
    current_item = items_separated_by_separator;
    for (i = 0; i < count; ++i) {
        iter = current_item;
        while (*iter && *iter != separator) iter++;
        length = (int)(iter - current_item);
        if (i == selected)   <----- obviously 'selected' should be integer 
            break;
        current_item = iter + 1;
    }
```

And there is much more examples inside this module. Am i right with my suspicions?

**Comment by dumblob** [comment]
You're right, this is an oversight. @aganm could you please take a look at it once more? We should sort this out ASAP. @exchg would you want to help?

**Comment by exchg** [comment]
Yep, i'll take a look tomorrow.

**Comment by aganm** [comment]
Thanks for reporting that.

#199

**Comment by dumblob** [comment]
Fixed in #199 .
